app/components/Onboarding/Onboarding.js

```diff
@@ -38,12 +46,43 @@ const Onboarding = ({
 }) => {
   const renderStep = () => {
     switch (step) {
+      case 'connection.type':
+        return (
+          <FormContainer
+            title='1. Ligntning Connection'
+            description='How will you connect to the Lightning Network?'
+            back={null}
+            next={() => changeStep(connectionType === 'local' ? 1 : 'connection.details')}
```

Comment:
hmmmmm. i don't like the fact that `steps` are now being stored as a string or a number. strings may be cleaner, but we should pick one. i'd say change `connection.details` and `connection.type` to numbers for now and we can get a separate PR do convert to strings after this is merged